gravitee-apim-console-webui/src/management/api/endpoints-v4/backend-services/endpoint-group/api-endpoint-group.harness.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ComponentHarness } from '@angular/cdk/testing';
+import { MatButtonHarness } from '@angular/material/button/testing';
+import { MatTabHarness } from '@angular/material/tabs/testing';
+import { MatInputHarness } from '@angular/material/input/testing';
+import { GioSaveBarHarness } from '@gravitee/ui-particles-angular';
+
+export class ApiEndpointGroupHarness extends ComponentHarness {
+  static hostSelector = 'api-endpoint-group';
+
+  private readInputValue = (input: MatInputHarness) => input.getValue();
```

Comment:
These private anon functions could be placed in a shared component

Reply:
Not for this scope if reused in another feature then shared component will be added